book/src/example.md

```diff
@@ -3,136 +3,26 @@
-> `customresourcedefinition.apiextensions.k8s.io/csinodeinfos.csi.storage.k8s.io created`
-
-### Create RBAC rules for CSI provisioner
-
-> $ kubectl create -f [https://raw.githubusercontent.com/kubernetes-csi/external-provisioner/1cd1c20a6d4b2fcd25c98a008385b436d61d46a4/deploy/kubernetes/rbac.yaml](https://github.com/kubernetes-csi/external-provisioner/blob/1cd1c20a6d4b2fcd25c98a008385b436d61d46a4/deploy/kubernetes/rbac.yaml)
-> ```
-> clusterrole.rbac.authorization.k8s.io/external-provisioner-runner created
-> clusterrolebinding.rbac.authorization.k8s.io/csi-provisioner-role created
-> role.rbac.authorization.k8s.io/external-provisioner-cfg created
-> rolebinding.rbac.authorization.k8s.io/csi-provisioner-role-cfg created
-> ```
-
-### Create RBAC rules for CSI attacher
-
-> $ kubectl create -f [https://raw.githubusercontent.com/kubernetes-csi/external-attacher/9da8c6d20d58750ee33d61d0faf0946641f50770/deploy/kubernetes/rbac.yaml](https://github.com/kubernetes-csi/external-attacher/blob/9da8c6d20d58750ee33d61d0faf0946641f50770/deploy/kubernetes/rbac.yaml)
-> ```
-> serviceaccount/csi-attacher created
-> clusterrole.rbac.authorization.k8s.io/external-attacher-runner created
-> clusterrolebinding.rbac.authorization.k8s.io/csi-attacher-role created
-> role.rbac.authorization.k8s.io/external-attacher-cfg created
-> rolebinding.rbac.authorization.k8s.io/csi-attacher-role-cfg created
-> ```
-
-### Create RBAC rules for node plugin
-
-Only the `driver-registrar` interacts directly with Kubernetes, so it's those RBAC rules that are needed:
-
-> $ kubectl create -f [https://raw.githubusercontent.com/kubernetes-csi/driver-registrar/87d0059110a8b4a90a6d2b5a8702dd7f3f270b80/deploy/kubernetes/rbac.yaml](https://github.com/kubernetes-csi/driver-registrar/blob/87d0059110a8b4a90a6d2b5a8702dd7f3f270b80/deploy/kubernetes/rbac.yaml)
-> ```
-> serviceaccount/csi-driver-registrar created
-> clusterrole.rbac.authorization.k8s.io/driver-registrar-runner created
-> clusterrolebinding.rbac.authorization.k8s.io/csi-driver-registrar-role created
-> ```
-
-### Create RBAC rules for CSI snapshotter
-
-The CSI snapshotter is an optional sidecar container. You only need to create these
-RBAC rules if you want to test the snapshot feature.
-
-> $ kubectl create -f [https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/01bd7f356e6718dee87914232d287631655bef1d/deploy/kubernetes/rbac.yaml](https://github.com/kubernetes-csi/external-snapshotter/blob/01bd7f356e6718dee87914232d287631655bef1d/deploy/kubernetes/rbac.yaml)
-> ```
-> serviceaccount/csi-snapshotter created
-> clusterrole.rbac.authorization.k8s.io/external-snapshotter-runner created
-> clusterrolebinding.rbac.authorization.k8s.io/csi-snapshotter-role created
-> ```
-
-### Deploy driver-registrar and hostpath CSI plugin in DaemonSet pod
+CRDs are created optionally `CSIDriverRegistry` and `CSINodeInfo`:
```

Comment:
I don't know if we need this anymore considering we're getting rid of the CRDs in 1.14.

Reply:
Let's remove CRD stmt from here, it does not add much value, glimpse into deployment example shows CRDs are inside if stmt, thus optional